### ISSUE TITLE
Keyboard Focus is not (or not clearly) visible #UX #Ergonomics #Accessibiliy

### ISSUE THREAD

**Issue by OLLI-S** [bitwarden/web#713]
## Describe the Bug
In the Web Vault I checked if the keyboard focus is clearly visible on all focusable elements.
This issue is related to #712 (Elements are not Accessible with the Keyboard).

### Unlock Vault (enter master password)
The keyboard focus is not visible at all at the "Show Password" icon:
![image](https://user-images.githubusercontent.com/8037790/99904408-e613b800-2cca-11eb-94df-8014d5850494.png)

At the button "Logout" the keyboard focus is visible, but it is not clearly visible:
![image](https://user-images.githubusercontent.com/8037790/99904769-6fc48500-2ccd-11eb-971a-c1991b721113.png)

Please use a blue border (like you have it on other elements):
![image](https://user-images.githubusercontent.com/8037790/99904759-51f72000-2ccd-11eb-87bd-ad2bf7d54654.png)

### Enter 2FA Code
The focus on the "Cancel" button is not clearly visible:
![image](https://user-images.githubusercontent.com/8037790/99908273-fb481100-2ce1-11eb-922f-ba822af3c70a.png)

### My Vault
In My Vault the keyboard focus is not clearly visible at the "Options" button:
![image](https://user-images.githubusercontent.com/8037790/99904653-81f1f380-2ccc-11eb-822a-bdf38fbedb86.png)

Please use a blue border, like you have it on the "Add Item" button.
![image](https://user-images.githubusercontent.com/8037790/99904676-ad74de00-2ccc-11eb-9f1d-8a7751700245.png)

Same for the "Options" button that is shown behind each entry in my vault (focus is not clearly visible):
![image](https://user-images.githubusercontent.com/8037790/99904691-e01ed680-2ccc-11eb-9780-742d654b431d.png)

At all other elements the keyboard **focus is clearly visible**.

### Adding / Editing an entry
When adding/editing an entry, the keyboard focus is not visible at all at the following elements:

Icon "Copy Verification Code": 
![image](https://user-images.githubusercontent.com/8037790/99908753-8f1adc80-2ce4-11eb-9572-8e57a3c3185b.png)

Icon "Remove URI":
![image](https://user-images.githubusercontent.com/8037790/99908620-ec625e00-2ce3-11eb-8dbd-70a87c7f89e3.png)

Icons "Remove" and "Drag to sort item":
![image](https://user-images.githubusercontent.com/8037790/99908584-cb017200-2ce3-11eb-975f-89500563855e.png)

Icon "Favorite":
![image](https://user-images.githubusercontent.com/8037790/99908682-38150780-2ce4-11eb-8873-acf4a5656761.png)

At the following icons the keyboard focus is not displayed correctly (the bottom line of the focus is missing):
![image](https://user-images.githubusercontent.com/8037790/99909380-77ddee00-2ce8-11eb-9efb-d20ed48c7731.png)
![image](https://user-images.githubusercontent.com/8037790/99909397-8debae80-2ce8-11eb-8e20-89fd909f6a51.png)

At the "Cancel" button the keyboard focus is not clearly visible:
![image](https://user-images.githubusercontent.com/8037790/99909440-d99e5800-2ce8-11eb-84a1-61c6b032ba6d.png)

### Tools
In the tools I discovered that the keyboard focus on the menu items is not always displayed as rectangle.
At the first menu item the focus is **displayed correctly**:
![image](https://user-images.githubusercontent.com/8037790/99909020-48c67d00-2ce6-11eb-83b4-459191185059.png)

At the second menu entry, the focus is only shown at the left side and the right side:
![image](https://user-images.githubusercontent.com/8037790/99909036-5f6cd400-2ce6-11eb-8666-f3db63c95795.png)

Here the bottom line of the focus indicator is missing (this problem occurs on most of the following menu items):
![image](https://user-images.githubusercontent.com/8037790/99909063-87f4ce00-2ce6-11eb-9ef9-daa3e278253d.png)

In the password generator the keyboard focus is not clearly visible at the buttons "Copy Password" and "Password History":
![image](https://user-images.githubusercontent.com/8037790/99909252-a3140d80-2ce7-11eb-8d9d-53404abce950.png)
![image](https://user-images.githubusercontent.com/8037790/99909271-c343cc80-2ce7-11eb-951b-e90abfa09203.png)
Also here a blue frame instead of a grey frame wold indicate the focus much better.

### Settings
In the Settings I checked every single page (I did **not yet check the Organizations**).
Also here you have the issue, that the keyboard focus in the navigation menu is often truncated (first entry is OK):
![image](https://user-images.githubusercontent.com/8037790/99909650-1ae33780-2cea-11eb-8708-9ea88359197c.png)
![image](https://user-images.githubusercontent.com/8037790/99909683-45cd8b80-2cea-11eb-9917-3eebab260c12.png)
![image](https://user-images.githubusercontent.com/8037790/99909745-98a74300-2cea-11eb-8900-8be7c3da1b83.png)

At the page "Premium membership" the keyboard focus is not clearly visible at the buttons "Reinstate Subscription" and "Download License":
![image](https://user-images.githubusercontent.com/8037790/99909809-ed4abe00-2cea-11eb-946a-a34b3dadd078.png)
![image](https://user-images.githubusercontent.com/8037790/99909823-fcca0700-2cea-11eb-8742-e06a164eaac5.png)
Did I mention that a blue frame is better....? :wink:

At the page "Billing" the keyboard focus is not clearly visible at the buttons "Add Credit" and "Change Payment Method":
![image](https://user-images.githubusercontent.com/8037790/99909964-c3de6200-2ceb-11eb-80b2-5cbaab8c7e93.png)
![image](https://user-images.githubusercontent.com/8037790/99909978-d35dab00-2ceb-11eb-91ee-df4abb940b34.png)

At the page "Two-step Login" there are also buttons having a grey focus indicator:
![image](https://user-images.githubusercontent.com/8037790/99909995-fd16d200-2ceb-11eb-88cf-df880795cbba.png)
![image](https://user-images.githubusercontent.com/8037790/99910002-0a33c100-2cec-11eb-9c0d-045e9136665f.png)
This occurs on all listed providers.

And last at the page "Domain Rules" there is also a button with a grey focus indicator:
![image](https://user-images.githubusercontent.com/8037790/99910077-58e15b00-2cec-11eb-94d4-02cbbfde4469.png)

## Steps To Reproduce
Just use the TAB key to cycle through the elements

## Expected Result
The keyboard focus must be clearly visible on all interactive elements (according to WCAG 2.1, chapters [2.4.7](https://www.w3.org/TR/WCAG21/#focus-visible) and [3.2.1](https://www.w3.org/TR/WCAG21/#on-focus)).

## Environment
- Operating system: Windows 10 Pro 64-Bit Version 20H2 (Build 19042.630)
- Browser: Microsoft Edge Chromium 87.0.664.41 (Offizielles Build) (64-Bit)
- Build Version: 2.17.0

**Comment by eliykat** [comment]
Thanks @OLLI-S!
